x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -75,21 +41,12 @@ export function ServiceMap({ serviceName }: ServiceMapProps) {
       return callApmApi({
         pathname: '/api/apm/service-map',
         params: {
-          query: {
-            ...params,
-            start,
-            end,
-            uiFilters: JSON.stringify(params.uiFilters)
-          }
+          query: { ...params, start, end, uiFilters: '' }
```

Comment:
Let's be explicit here (instead of spreading):
```suggestion
          query: { serviceName, environment, start, end }
```

Reply:
Will do that in PR for UI filters changes.